src/util.ts

```diff
@@ -42,17 +42,13 @@ export function removeAllPatterns(str: string, patterns: string[]): string {
  * case of the path.
  *
  * @param p The input path
- * @param normalize_case Whether we should normalize the case of the path
  * @returns The normalized path
  */
-export function normalizePath(p: string, normalize_case = true): string {
+export function normalizePath(p: string): string {
```

Comment:
Why remove path case normalization for Windows?

Reply:
Is was a bugfix for some test in suite "Diagnostics". 
I replaced the source directory from config by a `dummyPath` to make it independent from the config. It is out of scope of the test. 
But the test failed because on windows, the path "dummyPath" is translated to "dummypath/..." by the normalize_case parameter. I did not see the need. 

Was there a error case which makes it necessary?

Reply:
In general, not normalizing paths when you work on a filesystem with normalized paths can cause trouble. I was seeing issues where I'd generate two different file URIs and VSCode would treat them as two files, even though they actually referred to the same file.

I'd rather do the `normalizePath` on the test value just to be safe.

Reply:
I have revert the modification and update test.